opm/input/eclipse/EclipseState/Grid/Keywords.hpp

```diff
@@ -73,9 +81,15 @@ struct keyword_info {
 
     keyword_info<T>& global_kw(bool g) {
         this->global = g;
+        if(g)
+            this->local_in_schedule = false;
         return *this;
     }
 
+    keyword_info<T>& global_kw_until_edit() {
+        this->global = this->local_in_schedule = true;
```

Comment:
While correct code, it is easy to misread in a hurry. I would very much prefer a single assignment per line.